integreat_cms/cms/forms/machine_translation_form.py

```diff
@@ -125,7 +129,7 @@ def save(self, commit=True, foreign_form_changed=False):
 
```

Comment:
```suggestion

        language_nodes = self.cleaned_data["mt_translations_to_create"].union(
            self.cleaned_data["mt_translations_to_update"]
        )
```
What do you think about pulling out the language nodes from the inner block and only check the hix score if there are language nodes to begin with?

Reply:
I wanted to use the `mt_is_enabled()` method to check whether the MT form is shown on the page or not, see e.g. https://github.com/digitalfabrik/integreat-cms/blob/df2863ec4d47ce7f37aab2d8fdd4564514047431/integreat_cms/cms/templates/pages/page_form_sidebar/minor_edit_box.html#L21

The problem if we also use this method whether something was selected, is, that it will e.g. hide the form if MT was selected and the form validation of the page form or page translation form fails, the form is directly rendered again (with the error message indicating which form field caused the error), but the MT form will just be hidden in this case:

![Screenshot 2023-04-17 at 11-08-12 Integreat Editorial System](https://user-images.githubusercontent.com/16021269/232438789-db3b3a6e-fc75-48c4-8da8-efe103196edd.png)

So I think it would make sense to separate these two steps (showing the form and having something selected)... Maybe to reduce the potential for misunderstandings, we could adapt the docstring of the `mt_is_enabled()` method?

Reply:
Or we could rename `mt_is_enabled()` to `show_mt_form` or similar? I guess I just wanted to make it sound similar to the inbuilt `is_valid()`, but if the meaning isn't clear, it's not a good choice.

Reply:
Ahh, I see. Yes, I think renaming it would be a good idea, from the naming and logic inside it I was assuming this was just used to check if MT had been enabled (by the user) for this page.

How about `mt_form_is_enabled` as the method name and "Helper method to decide if this form should be shown, or if it should be hidden for the current language due to a lack of MT-compatible child language nodes"  or something?

Reply:
Yes, sounds good! :+1: